### PR TITLE
[ADD] new module to realize mass actions (confirm, ...) on sale orders;

### PR DESCRIPTION
**Pull request by legalsylvain**
Add new modules that simply add multi action to realize mass action on sale orders;
For the time being :
- possibility to confirm many quotations at a time ;

(possible improvements)

Thanks for your review.

### CODE REVIEW

File: sale_order_mass_action/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    'version': '1.0',
+    'author': "GRAP,Odoo Community Association (OCA)",
+    'website': 'http://www.grap.coop',
+    'license': 'AGPL-3',
+    'category': 'Sales',
+    'description': """
+Sale Orders Mass Action
+========================
+
+Facilities to realize mass action on Sale Orders:
```
Comment (pedrobaeza) [inline sale_order_mass_action/__openerp__.py]: s/realize/perform

File: sale_order_mass_action/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+* A wizard which allows on multiple Sale Orders at a time to:
+    * confirm Sale Orders;
+
+Possible Improvements:
```
Comment (pedrobaeza) [inline sale_order_mass_action/__openerp__.py]: Put in Know issues / Roadmap, and follow OCA's template for the rest of the usual sections: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by eLBati** [comment]
:+1: thanks

**Comment by legalsylvain** [comment]
@pedrobaeza, could you approve this PR if all is OK for you now, I'd like to port this module in V8 / V9 after, cherry picking commits.
thanks.

**Comment by legalsylvain** [comment]
ping @pedrobaeza. any news on that review ? 
Regards.

**Comment by bealdav** [comment]
:+1:  thanks

**Comment by legalsylvain** [comment]
Thanks for your review !

**Pull request merged**